### ISSUE TITLE
Accessibilite_2-Segmentation

### ISSUE THREAD

**Issue by stecau** [stecau/DevWeb-Projet4#5]
Problème : Pas de bonne segmentation des sous parties avec titre h2, … et avec mot-clé ou synonyme

- Explications : Utilisation d’une image plutôt que de mettre un titre h2 qui casse le segmentation de la page et peut être également assimilé comme du ‘Black Hat’
- Bonne pratique : Mettre un titre H2 plutôt qu’une image et respecter une segmentation logique de page.
- A Corriger/Faire : 

- [x] Remplacer les images correspondant à des titres par des balises h2
